src/js/flows/submitSearch.js

```diff
@@ -26,9 +26,16 @@ export default function submitSearch(
 
     const state = getState()
 
+    let view = document.getElementsByClassName("view")[0]
+
     if (save) {
```

Comment:
Looking at this file made me realize we may have a bug in the code. If I submit a search with `save: false` it will not update the `History` state slice (which is what is used in the back and forward buttons) or the `Investigation` state slice (which is used for the history in the left sidebar). The main place we use `save: false` is when a user clicks on a previous search in the left sidebar.  When they click a previous search, it will update the searchbar and the viewer, but will not add an entry to the history. This results in:

```
change search input to "a",  hit enter, view results
change search input to "b",  hit enter, view results
click on a previous search in the left side bar, view results
click the "back" button

I'd expect the search input to be "b" but it is "a"
```

To fix this, here's what I think we should do.
* Always run `History.push(record, time.toTs())`
* Only run `Investigation.push(record, time.toTs())` if save === true.

As for making sure that the history entry has the correct scroll position, see my comment about adding a `History.update` call in the onScrollEnd callback in the `Viewer` component.

Note:
There may be a case when we do in fact not want to save to the History or the Investigation. Like when clicking on the "back" and "forward" buttons. If this appears to be true, then we may need to change the save argument consider both object like: `submitSearch({history: false, investigation: false})`.